configure.ac

```diff
@@ -190,7 +190,9 @@ CHECK_CMA(
 AM_CONDITIONAL([USE_CMA], [test "$transport_cma" = "yes"])
 
 AS_IF([test "transport_xpmem" = "yes" -o "$transport_cma" = "yes"],
-      [AC_DEFINE([USE_ON_NODE_COMMS], [1], [Define if any on-node comm transport is available])])
+      [AC_DEFINE([USE_ON_NODE_COMMS], [1], [Define if any on-node comm transport is available])
+       AC_DEFINE([ENABLE_HARD_POLLING], [1], [Enable hard polling])
```

Comment:
currently, this will then force all the wait_until calls to use the variable polling option, but don't we only want this when its a shared memory case?